README.md

```diff
@@ -126,6 +127,14 @@ Defaults to the AWS Organization default, `OrganizationAccountAccessRole`.
 An IAM role session name that will be passed to each Cove session's `sts.assume_role()` call. 
 Defaults to the name of the role being used if unset.
 
+`policy`: Optional[str]
+
+A policy document that will be used as a session policy in each Cove session's `sts.assume_role()` call. Unless the value is None, it is passed through via the Policy parameter.
+
+`policy_arns`: Optional[List[PolicyArn]]
```

Comment:
`PolicyArn` may not make much sense here. Should I clarify in the documentation? Or should I replace `PolicyArn` everywhere with `List[Dict[str, str]]`?

I wanted to enforce the `arn` key in the type system, but maybe it's not worth it if it makes the documentation difficult.

Reply:
I revered it back to a str just for clarity, boto3 will validate this at runtime and blow up if it's wrong anyway which I guess is the benefit of our previous challenge :)